painting/src/main/java/com/ataulm/artcollector/painting/ui/PaintingActivity.kt

```diff
@@ -27,12 +34,46 @@ class PaintingActivity : AppCompatActivity() {
         val paintingId = intent.data!!.pathSegments.last()
         injectDependencies(PaintingId(paintingId))
 
+        // TODO: check with app links that it still works and doesn't care about postponed transitions
+        postponeEnterTransition()
```

Comment:
on the PaintingActivity side, we can use `postponeEnterTransition` to _freezeframe_. This will stop the transition on the first frame, so you'll see the GalleryActivity with the selected painting ImageView (pop in front of the artist label) ready to transform.

but we want to wait until the image is loaded on this side, else it'll jump from image -> imageView background color -> image while it transforms.

**it's important to never ever call `postponeEnterTransition()** without calling `startPostponedEnterTransition()` else your app will look like it's frozen.